source/GameInterface/Services/Settlements/Patches/GarrisonWagePaymentLimitSettlementPatch.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Common;
+using Common.Messaging;
+using Common.Util;
+using GameInterface.Policies;
+using GameInterface.Services.Settlements.Messages;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using TaleWorlds.CampaignSystem.Settlements;
+
+namespace GameInterface.Services.Settlements.Patches;
+
+/// <summary>
+/// Handle changes for GarrisonWagePaymentLimit
+/// </summary>
+[HarmonyPatch(typeof(Settlement))]
+public class GarrisonWagePaymentLimitSettlementPatch
+{
+    [HarmonyPatch(nameof(Settlement.GarrisonWagePaymentLimit), MethodType.Setter)]
+    [HarmonyPrefix]
+    private static bool GarrisonWagePaymentLimitPrefix(ref Settlement __instance, ref int value)
+    {
+        if (AllowedThread.IsThisThreadAllowed()) return true;
+        if (CallOriginalPolicy.IsOriginalAllowed()) return true;
+
+        if (ModInformation.IsClient) return false;
```

Comment:
Change this to match up to line 42 of https://github.com/Bannerlord-Coop-Team/BannerlordCoop/blob/development/source/GameInterface/Services/Armies/Patches/ArmyLifetimePatches.cs#L42